tests/integration/033_variable_set/main.tf

```diff
@@ -70,3 +70,9 @@ resource "env0_variable_set" "project_scope" {
     format = "text"
   }
 }
+
+resource "env0_variable_set_assignment" "assignment" {
+  scope    = "project"
+  scope_id = env0_project.project.id
+  set_ids  = var.second_run ? [env0_variable_set.org_scope.id] : [env0_variable_set.project_scope.id]
```

Comment:
the test seems a bit missing... can we check the outcome of that? using output or some other method, to see that the assignment doesn't exist on the org after the second run, and exists where it should after on each run?

Reply:
This is tested in the acceptance tests.
The integration tests are limited in their functionality and are more of a sanity just to make sure things really work...
I don't think there's an easy way to test what you're asking with the current integration tests... /:

Reply:
Alright. As long as there is a test for it